src/entry-client.js

```diff
@@ -24,11 +24,10 @@ import { getModuleName, safelyRegisterModule } from './utils';
  *
  * @param   {object} c             Vue component definition object for destination route
  * @param   {object} fetchDataArgs Context argument passed to fetchData
- * @param   {object} spruDefaults  Defaults from initializeClient
  * @returns {boolean}              True if we should process this route update through the
  *                                 fetchData/middleware pipeline
  */
-function shouldProcessRouteUpdate(c, fetchDataArgs, spruDefaults) {
+function shouldProcessRouteUpdate(c, fetchDataArgs) {
```

Comment:
defaults are just inlined now since we don't need to be able to adjust them via a flag from the client apps